app/lib/services/eligibility/tsf_calculator.rb

```diff
@@ -0,0 +1,61 @@
+module Services
+  module Eligibility
+    class TsfCalculator
```

Comment:
With the two types of funding coming through this would a more top level name make it clearer that it provides info on both targeted support funding and targeted delivery funding?

`Services::Eligibility::TargetedFundingCalculator`? 

Initially I spent a bit thinking this one worked exclusively on targeted support funding and was confused about the presence of targeted_delivery_funding as a key.

Also, I'm a bit unsure of introducing a new acronym here (tsf), is there a reason not to spell it fully out as targeted_support_funding like how targeted_delivery_funding is fully spelled out?